src/geo/gmaps/gmaps-default-map-style.js

```diff
@@ -1,69 +0,0 @@
-var DEFAULT_MAP_STYLE = [{
```

Comment:
But I feel we should take into account when the user wants to create a map from the beginning using just the CARTO.js library, shouldn't we?